SpywareFilter/sections/specific.txt

```diff
@@ -52,6 +52,7 @@
 ||lazimbana.com/product/hit-increment/
 ||rusvideos.tv/rusvideos-site/analytics.js
 ||pla.naver.com/log/
+/AppMeasurement.js$script,domain=channel4.com
```

Comment:
$script is redundant here, also add:

/AppMeasurement_Module_ActivityMap.js$domain=channel4.com